foundry/foundry.py

```diff
@@ -725,9 +740,16 @@ def _load_data(self, file=None, source_id=None, globus=True):
                 )
             except:
                 # Try to read individual lines instead
-                self.dataset.dataframe = pd.read_json(
+                try:
```

Comment:
instead of nesting the try/except clauses, I'd recommend chaining them, as shown in this section: https://docs.python.org/3/tutorial/errors.html#handling-exceptions